diff --git a/node_modules/grunt-favicons/tasks/favicons.js b/node_modules/grunt-favicons/tasks/favicons.js
index a5301d7..85216a9 100644
--- a/node_modules/grunt-favicons/tasks/favicons.js
+++ b/node_modules/grunt-favicons/tasks/favicons.js
@@ -449,7 +449,7 @@ export default function(grunt) {
                 // Cleanup
                 if (options.regular) {
                     ['16x16', '32x32', '48x48'].forEach(function(size) {
-                        fs.unlink(path.join(f.dest, size + '.png'));
+                        fs.unlink(path.join(f.dest, size + '.png'), () => {});
                     });
                 }

